### DISCUSSION TITLE
Aphrodite boss battle

### DISCUSSION THREAD

**Discussion by Derphead** [ErinaSugino/Starbound-SBR-Mod#108] [Trouble Shooting] [answered]
So, in the boss battle against aphrodite, everything works fine untill the second phase (second time you get to bone her) when you interact with the table she and you dissapear and the position wheel doesn't show up. I can still move out of the node, but not progress. And Aphrodite doesn't appear anymore.

**Comment by ErinaSugino** [comment]
Log file.

**Comment by Derphead** [comment]
[starbound.log](https://github.com/ErinaSugino/Starbound-SBR-Mod/files/15444393/starbound.log)

**Reply by ErinaSugino** [reply]
If you go into `/positions/twoactors/reverse_cowgirl` and inside `reverse_cowgirl.config` fix the wrong spelling of the dialog paths (like in [here](https://github.com/ErinaSugino/Starbound-SBR-Mod/commit/12d7649b77b7f718259aa8a6b21ea5bd19f36884)), does the fight work then?

**Reply by Derphead** [reply]
where do I find that?

**Reply by ErinaSugino** [reply]
In the SBR mod

**Reply by Derphead** [reply]
I'm not sure if I'm blind, but I can't find a reversse_cowgirl.config file, or the indicated misspelled lines in any of the config files in Starbound\mods\sexbound-reborn-config. 
This is my first time dealing with mods so I'm kinda confused :D

**Reply by ErinaSugino** [reply]
Not in any normal config files. Inside the mods content.
If you have the pak version you will need to unpak that or get the source version. I gave you the exact filepath inside the mod assets.

**Reply by Derphead** [reply]
Still doesn't work. I even tried changing it to reverse_cowgirl instead of reversecowgirl like you did in the link, just in-case it was a typo aswell, since I saw several other mentions in configs about reverse_cowgirl. Aphrodite still dissapears when initiating the second round of sex.

**Reply by Derphead** [reply]
Tried with a different character incase it was something to do with my Neki but she still dissapears.
[starbound.log](https://github.com/ErinaSugino/Starbound-SBR-Mod/files/15467891/starbound.log)

**Reply by ErinaSugino** [reply]
I'll need you to get me another log, but this time enable "debug" and "info" level logging, in the main sexbound.config file, near the very bottom (NOT "behavior"). I'd also like you to type in chat before each interaction, to know when what happens. This seems to be a deeper issue.

**Reply by Derphead** [reply]
Here's the log. I wrote INTERACTION1 and INTERACTION2 so its easy to find :)
[starbound.log](https://github.com/ErinaSugino/Starbound-SBR-Mod/files/15476298/starbound.log)

**Comment by ErinaSugino** [comment, marked as answer]
Apply the changes of this commit and see if this fixes it (and doesn't break anything else.)
https://github.com/ErinaSugino/Starbound-SBR-Mod/commit/420186e23b8ec72e7955254e3560fd46411b6dfa

**Reply by Derphead** [reply]
It worked! Thank you very much for your help :)
I'll let you know if it broke something else, but for now atleast it doesn't appear so.
